dp-game-webapp/app/game/validate/page.tsx

```diff
@@ -106,57 +100,185 @@ export default function Validate() {
               </div>
             ) : (
               <>
-                <div className="mt-6 mb-6 flex-col items-center justify-between">
-                  <b> Very bad campaign</b> (1 out of a 100)
-                  <br />
-                  {Math.round(percentiles[0]).toLocaleString()} Conversions
-                </div>
-
-                <div className="mt-6 mb-6 flex-col items-center justify-between">
-                  <b> Bad campaign</b> (1 out of a 10)
-                  <br />
-                  {Math.round(percentiles[1]).toLocaleString()} Conversions
-                </div>
-
-                <div className="mt-6 mb-6 flex-col items-center justify-between">
-                  <b> Average campaign</b> (5 out of a 10)
-                  <br />
-                  {Math.round(percentiles[2]).toLocaleString()} Conversions
-                </div>
-
-                <div className="mt-6 mb-6 flex-col items-center justify-between">
-                  <b> Good campaign</b> (1 out of a 10)
-                  <br />
-                  {Math.round(percentiles[3]).toLocaleString()} Conversions
-                </div>
+                <Percentiles
+                  percentiles={percentiles}
+                  impressions={impressions}
+                  className=""
+                />
+                <AdjustVariance
```

Comment:
We should also add CampaignStats on this page so that the user can check what average values were set